jobs/FunctionTest/FunctionTest.groovy

```diff
@@ -336,7 +355,7 @@ def ovaPostTest(TESTS, ova_stash_name, ova_stash_path, repo_dir, test_type){
 
 def manifestTest(TESTS, manifest_stash_name, manifest_stash_path, repo_dir, test_type){
     setManifest(manifest_stash_name, manifest_stash_path)
-    test_stack = "-stack vagrant"
+    test_stack = "-stack local_run"
```

Comment:
Thanks!
I created a [PR](https://github.com/RackHD/RackHD/pull/802) to add a stack ```docker_local_run```  to rackhd/test/config/stack-config.json
I didn't use the existing stack ```docker``` because there are some differences , such as the port of mongo.

I'm curious about the usage of stack ```docker```.
It's for docker post test?
If so, the port should be the same with the port of my PR.
Because I'm using the same port config with our docker post test.